model/src/main/java/io/syndesis/model/filter/FilterStep.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.model.filter;
+
+import io.syndesis.model.integration.Step;
+
+import java.util.List;
+
+//@Value.Immutable
+//@JsonDeserialize(builder = FilterStep.Builder.class)
```

Comment:
Yes! 

I had to comment it out, or else build fails. It seems that Immutables doesn't fancy builder hierarchies.